Specs/Core/getAbsoluteURLSpec.js

```diff
@@ -0,0 +1,24 @@
+/*global defineSuite*/
+defineSuite([
+        'Core/getAbsoluteURL'
+    ], function(
+        getAbsoluteURL) {
+    "use strict";
+
+    it('works as expected', function() {
+        var result = getAbsoluteURL('http://www.mysite.com/awesome?makeitawesome=true');
+        expect(result).toEqual('http://www.mysite.com/awesome?makeitawesome=true');
+
+        result = getAbsoluteURL('http://www.mysite.com/somefolder/awesome.png#makeitawesome');
+        expect(result).toEqual('http://www.mysite.com/somefolder/awesome.png#makeitawesome');
+
+        result = getAbsoluteURL('awesome.png');
+        expect(result).toEqual('http://localhost:8080/Specs/awesome.png');
+    });
```

Comment:
I think these tests are a good start, but could be improved a bit. Here are some cases it should handle:

* Pass in a url that is already absolute (that's what your first two tests are doing, you can remove one of them)
* Pass in a relative url. Your last test is testing this okay, but to be a little safer you should check that the result equals something like `document.location.href + 'awesome.png'` instead of hard-coding localhost:8080.
* Pass in a relative url and a base url, the result should be combined.